UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Webjob.UnitTests/Services/FulfilmentDataServiceTest.cs

```diff
@@ -81,19 +90,6 @@ private SalesCatalogueProductResponse GetSalesCatalogueResponse()
         }
         #endregion
 
-        [Test]
```

Comment:
It seems this has been removed accidentally. Kindly confirm about the same.

Reply:
Yes again it has been added